### ISSUE TITLE
The Other 10

### ISSUE THREAD

**Issue by meneal** [LambdaConglomerate/x9115lam#43]
This is just an issue for starting a list of potential papers for the other 10 references we need for the final paper.  I like the idea of a wiki, but this was just faster.

**Comment by meneal** [comment]
One potential paper is listed in the TCA paper by Nam:

[A survey on transfer learning, Pan et al](http://www1.i2r.a-star.edu.sg/~jspan/publications/TLsurvey_0822.pdf)

I just mention this one since I have no real background for transfer learning and it would certainly fill in some gaps for the TCA paper.

**Comment by aisobran** [comment]
Can someone link the wiki?

Has anyone added this paper?
http://fengzhang.bitbucket.org/publications/msr2014_universal_model.pdf

It references the TCA paper.

**Comment by ghost** [comment]
Sorry, I forgot to create the wiki page. [Here's the link](https://github.com/LambdaConglomerate/x9115lam/wiki/References-for-Final-Paper).

**Comment by aisobran** [comment]
After reviewing for a while, I think we could split up more categories, either for each paper or the main sections as each paper usually has the same setup:

Feature Selection: Many papers used different features

Machine Learning Algorithms: A review of the machine leaning algorithms used

Optimizations:

Data Sources: Sourceforge, githhub, ..

Metrics: How success was assessed

There may be more. 

Now to break this up we could do something like this:

**Paper 1:**
Features: Explanation with criticisms
ML:
Sources:
Metrics:
Overall Success:

**Paper 2:**
.....

**Future Work:**
Features: Out best ideas
ML:
Sources:
Metrics:

Or 

**Features:**
Paper1: features used in this paper with criticisms
Paper2:
....
Future Work: What features we think are important

**ML:**
Paper1: ML used in this paper with criticisms
Paper2: ...
...Future Work:

**Source:**

## **Metrics:**

**Comment by aisobran** [comment]
This may also give a framework to pull what we need out and put it together coherently.

**Comment by aisobran** [comment]
Are you guys sure you don't want to do markdown? I feel like it could save us time with formatting.

**Comment by meneal** [comment]
Latex is fine with me.  It already has been partially written in Latex so I
think it's fine to just stick with that.  It's so easy to do bibliography
related stuff in Latex too, and that's going to be half the battle with
this paper.

**Comment by ghost** [comment]
I agree with sticking with Latex for a paper like this. I don't think the formatting will take considerably longer than markdown.

**Comment by meneal** [comment]
I think the second option is better for sure as far as talking about the papers.  I think it would come out to be really choppy if we just discuss each paper individually.  Discussing each topic with all of the papers that apply to that topic is much better in my opinion.

I'm not totally sure on the categories yet.  I need to spend a bit of time going through the papers to think about categories again.  It might be useful too to look at some of these papers I linked in the wiki since they are lit reviews and that's basically what we're doing here.  Menzies says in his rubric that:

"Mention as many as possible of items listed 1 to 19, above."  Where those 1-19 were the feature extraction items like Motivational Statements.  We could try to work some of those feature extraction items into categories?

**Comment by meneal** [comment]
The most recent of those papers is the one by Radjenovic and they have the following categories in their paper:  

They actually split their paper into this results section where they discussed specific research questions, some of those were really things that we can use.  Like:  "RQ2.2: Are complexity metrics useful for fault prediction?...".

I guess we don't want to go too far into lit review since Menzies also states in the rubric that this is really a discussion of improvement or failure to improve research through time.  So that's an important way to look at this too.

**Comment by aisobran** [comment]
Lets separate it out by paper if we are going to separate it out by items 1-19. I'm not sure it would read well to have something like 8 motivational statements in a row from 8 different papers.

**Comment by aisobran** [comment]
That also lets us reuse what we've written so far.

**Comment by meneal** [comment]
I agree with you about motivational statements, but I think that motivational statements is a horrible category though.  Something like statistical tests would be a good category to look at and would be something that would make more sense to discuss across all of the papers.

**Comment by meneal** [comment]
I definitely agree though that anything we can keep from our summaries we should totally use.

**Comment by aisobran** [comment]
Of the 19 these could go together:
Stat tests, Sampling Procedures, Patterns, Anti-patterns
as well as a keyword list.

Pretty much everything else has to be said in the context of the paper.

**Comment by meneal** [comment]
I think that those three are fairly reasonable categories to look at using.  Do you think it would make sense to look at any of the other extractors as sections?

I'm just throwing out ideas, but another possibility would be doing something like coming up with really high level categories and then fleshing out those categories with the papers we've already read and then the rest of what we need in citations. For example what about something like cross-project defect prediction?  We have 2/8 papers that we worked on that look at cross project defect prediction.

**Comment by meneal** [comment]
Or something like Cost Effectiveness, since there was extensive discussion of cost effectiveness measures in these papers?

**Comment by aisobran** [comment]
Works for me.

**Comment by meneal** [comment]
How would it be if we just post another issue and then just come up with a list of categories and then just come to some agreement on a final list?

**Comment by ghost** [comment]
I like the splitting up with high level categories. Doing it that way may make it easier to find related papers.

**Comment by ghost** [comment]
I'm fine with posting an issue for deciding categories.
